readme.md

```diff
@@ -238,6 +238,7 @@
 - [Integration](https://github.com/stn1slv/awesome-integration#readme) - Linking together different IT systems (components) to functionally cooperate as a whole.
 - [Node-RED](https://github.com/naimo84/awesome-nodered#readme) - A programming tool for wiring together hardware devices, APIs, and online services.
 - [Low Code](https://github.com/zenitysec/awesome-low-code#readme) - Allowing business professionals to address their needs on their own with little to no coding skills.
+- [Clubhouse](https://github.com/ehsanghaffarii/awesome-clubhouse#readme) - A list of best tools, APIs, and resources for Clubhouse The social audio app.
```

Comment:
Should this be "the Clubhouse social app"?

Reply:
Yes, that's right 👍